crates/escrow/src/lib.rs

```diff
@@ -578,6 +578,13 @@ impl<T: Config> Pallet<T> {
         let last_point = <PointHistory<T>>::get(<Epoch<T>>::get());
         Self::supply_at(last_point, height)
     }
+
+    /// RPC only
+    pub fn total_locked() -> BalanceOf<T> {
```

Comment:
Incidentally, getting the actual total locked (staked) is something the lib also needs, but I don't see a way to get it without simply looping over all the locked stakes. Not sure if this is acceptable for an RPC call only (as it won't be used in any transactions) - or if it should just be left as lib functionality.

Reply:
The problem with this call is the potential for Denial of Service (DoS) but we anyway have looping operations in other RPC calls so I'm not sure. However, as mentioned in my other comment the best place for this functionality would be in squid.